Client/src/bkr/future/watchdog/extend.py

```diff
@@ -0,0 +1,81 @@
+# Copyright Contributors to the Beaker project.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+"""
+.. _bkr-watchdog-extend:
+
+bkr watchdog extend: Extend Beaker watchdog time
+================================================
+
+.. program:: bkr watchdog extend
+
+Synopsis
+--------
+
+|  :program:`bkr watchdog extend` [*options*] <taskspec>
+|       [:option:`--by` <seconds>]
+
+Description
+-----------
+
+Extends the watchdog time for one or more recipes by specifying one
```

Comment:
Does it "extend" the watchdog time? If the current watchdog is set for 30 minutes from now, if I "extend" it for 10 minutes is the new watchdog 40 minutes or 10 minutes?